content/background.html

```diff
@@ -58,21 +58,20 @@ <h3> Ethnicity</h3>
 
 <div class="vbx2"></div>
 
-<h3>Mother Tongue</h3>
+<h3>Sexual Orientation</h3>
 <p class="half">
   <span class="big">
-  45% of respondents at least speak two languages, with English not being their first language.
+  6.9% of respondents are part of the LGBTQ+ community.
   </span>
-  There was a significant number of students who spoke English and Mandarin as their first language, but overall,
-  there is a wide variety of language spoken by students.
+  4 students preferred not to disclose.
 </p>
 <div class="half second">
-  <span class="graph-title">Students' Mother Tongue</span>
-  <div id="mother-tongue"></div>
+  <span class="graph-title">Students' Sexual Orientation</span>
+  <div id="sexual-orientation"></div>
 </div>
 
 <div class="vbx2"></div>
-
+<h2> International students, Immigration and Home Location </h2>
```

Comment:
```suggestion
<h2> International Students, Immigration and Home Location </h2>
```
should it be entirely title caps?